respa_admin/static_src/styles/page-home.scss

```diff
@@ -120,10 +120,12 @@ main {
     }
   }
 
-  section.resources,
-  section.units {
+  .main-section {
+      .panel-body:before, .panel-body:after {
```

Comment:
Please change the indentation two 2 space here so it matches the file formatting